.github/workflows/build-image-frontend.yml

```diff
@@ -51,6 +51,22 @@ jobs:
             -   name: Checkout
                 uses: actions/checkout@a5ac7e51b41094c92402da3b24376905380afc29 # v4.1.6
 
+            # Copy legal information from root dir to frontend dir for publishing
+            -   name: Copy and paste legal information
+                run: |
+                    cp ./DEPENDENCIES_FRONTEND ./frontend/DEPENDENCIES
+                    cp ./LICENSE ./frontend/LICENSE
+                    cp ./NOTICE.md ./frontend/NOTICE.md
+                    cp ./SECURITY.md ./frontend/SECURITY.md
+
+            # Needed to create multi-platfrom image
+            -   name: Set up Docker Buildx
+                uses: docker/setup-buildx-action@d70bba72b1f3fd22344832f00baa16ece964efeb # v3.3.0
+
+            # Needed to create multi-platfrom image
+            -   name: Set up QEMU
```

Comment:
Removed. Beside that I needed to update my dockerfile as I download jq and that was currently only done for amd64. 

Now I run into this [issue](https://github.com/npm/cli/issues/4828) in my latest run. Do you have arm and amd related setups running in your team? Because it seems that the package-lock.json isn't always agnostic. 

Beside that I could try updating dependencies.

Reply:
We don't have anything arm and amd related setups running in the portal-frontend. One relevant difference might be that we execute the build in the workflow and not in the dockerfile:
https://github.com/eclipse-tractusx/portal-frontend/blob/v2.0.0-RC8/.github/workflows/release.yml#L84
https://github.com/eclipse-tractusx/portal-frontend/blob/v2.0.0-RC8/.github/workflows/release.yml#L50
https://github.com/eclipse-tractusx/portal-frontend/blob/v2.0.0-RC8/package.json#L105
https://github.com/eclipse-tractusx/portal-frontend/blob/v2.0.0-RC8/.conf/Dockerfile.prebuilt#L31